<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiObjectsTableWidget</class>
 <widget class="QWidget" name="UiObjectsTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTools" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="UiPushButton" name="btnBuckets">
        <property name="text">
         <string>桶列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="UiPushButton" name="btnRefresh">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="UiPushButton" name="btnUpload">
        <property name="text">
         <string>上传</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="UiPushButton" name="btnDownload">
        <property name="text">
         <string>下载</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>433</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="UiBreadWidget" name="widgetBread" native="true"/>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="UiPageWidget" name="widgetPage" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UiPushButton</class>
   <extends>QPushButton</extends>
   <header>src/fend/uicom/uipushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>UiPageWidget</class>
   <extends>QWidget</extends>
   <header>src/fend/uicom/uipagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UiBreadWidget</class>
   <extends>QWidget</extends>
   <header>src/fend/uicom/uibreadwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
